app/pages/ServiceListingPage/ServiceListingPage.tsx

```diff
@@ -26,10 +26,10 @@ import {
   Service,
 } from "../../models";
 import styles from "./ServiceListingPage.module.scss";
-import { algoliasearch } from "algoliasearch";
+import { searchClient } from "@algolia/client-search";
```

Comment:
**note:** This is lighter package for working just with search and index objects without bringing in other APIs for Recommend, Analytics, A/B testing, etc.... See: https://www.algolia.com/doc/libraries/javascript/v5/methods/search/.